test/unit_math_test.exs

```diff
@@ -88,6 +88,49 @@ defmodule ExDimensions.MathTest do
     end
   end
 
+  property :multiplcation_adds_unit_exponents do
```

Comment:
One potential advantage to using the simpler unit test instead of the property test is that if we only have the unit test then we won't need to execute `Enum.sort` within the `ExDimensions.Math` module or in these tests.

The reason why I added `Enum.Sort` on concatenated unit lists is that it gives us a consistent way to represent heterogenous lists of units. If we don't represent these lists consistently then tests will fail since something like `[Kilograms, Meters]` is not equal to `[Meters, Kilograms]`.

Reply:
One more thought:

If we wanted to keep the property test but we did NOT want to have to sort unit lists (which is pretty annoying and would be easy to forget), then we could overwrite the behavior of equality and comparison operators to treat unit lists that have the same elements in different orders as equal.

Now that I write this out, I'm realizing this is probably the way to go instead of scattering `Enum.sort` throughout the code. So I'll make that update.